app/components/Avatar.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from "react"
+import { Image, StyleProp, View, ViewStyle, ImageStyle, ImageSourcePropType } from "react-native"
+import { observer } from "mobx-react-lite"
+import { spacing } from "../theme"
+
+export type AvatarPresets = keyof typeof $viewPresets
+
+type CommonProps = {
+  /**
+   * An optional style override useful for padding & margin.
+   */
+  style?: StyleProp<ViewStyle>
+  /**
+   * An optional style to override the Image
+   */
+  imageStyle?: StyleProp<ImageStyle>
+}
+
+interface SingleAvatarProps extends CommonProps {
+  preset?: "workshop" | "talk"
+  /**
+   * The avatar to display
+   */
+  source: ImageSourcePropType
+}
+
+interface PanelAvatarProps extends CommonProps {
+  preset: "speaker panel"
+  /**
+   * Multiple avatars to display
+   */
+  sources: ImageSourcePropType[]
+}
+
+type AvatarProps = SingleAvatarProps | PanelAvatarProps
+
+/**
+ * Displays an avatar for a workshop, talk or speaker panel
+ */
+export const Avatar: React.FC<AvatarProps> = observer(function Avatar(props) {
+  const { style: $styleOverride, imageStyle } = props
+  const preset: AvatarPresets = $viewPresets[props.preset] ? props.preset : "workshop"
+  const $imageStyle = Object.assign({}, $viewPresets[preset], imageStyle)
+  const $containerStyle = [$baseContainerStyle, $styleOverride]
+
+  if (preset !== "speaker panel") {
+    return (
+      <View style={$containerStyle}>
+        <Image
+          source={(props as SingleAvatarProps).source}
+          style={$imageStyle}
+          resizeMode="contain"
+        />
+      </View>
+    )
+  } else {
+    const sources = (props as PanelAvatarProps).sources
+    return (
+      <View style={$containerStyle}>
+        {sources.map((source, index) => (
+          <Image
+            key={`panel-${index}-${source}`}
+            source={source}
+            style={[$imageStyle, { marginTop: spacing.small, marginLeft: -spacing.tiny * index }]}
```

Comment:
[nit] since `marginTop` doesn't change maybe it can be moved to `$imageStyle`

Reply:
Actually I noticed a discrepancy with the screenshot in sim vs the designs, we don't need to multiple by the index here. I extracted it to a $panelImageStyle and lift it from inline.